services/chainstorage/src/main/java/tech/pegasys/teku/services/chainstorage/StorageService.java

```diff
@@ -222,4 +236,30 @@ protected SafeFuture<?> doStop() {
   public ChainStorage getChainStorage() {
     return chainStorage;
   }
+
+  /** This method is called only on Ephemery network when reset is due. */
+  void resetDatabaseDirectories(final ServiceConfig serviceConfig) throws IOException {
+    final Path beaconDataDir = serviceConfig.getDataDirLayout().getBeaconDataDirectory();
+    final Path slashProtectionDir =
+        serviceConfig.getDataDirLayout().getValidatorDataDirectory().resolve("slashprotection");
+    deleteDirectoryRecursively(beaconDataDir);
+    deleteDirectoryRecursively(slashProtectionDir);
+  }
```

Comment:
now we probably want to setup some testing around this a little...

Reply:
Even if we just did a test around startup, or directly on this... also depending on how that washes out we'd probably put VisibleForTesting

Reply:
> now we probably want to setup some testing around this a little...

Not sure which of the files the test should go in.